### PR TITLE
refactor: action cell

### PR DESCRIPTION
**Pull request by ap0nia**
## Summary
Begin migrating section table body to MUIv5.

Section table body cells will be encapsulated in `apps/antalmanac/src/components/RightPane/SectionTable/cells`. This can be refactored later, but it will provide visual clarity on the current progress of refactoring and reduce the size of the main body file.

## Test Plan
Visually confirm that the component works the same as before.

### CODE REVIEW

File: apps/antalmanac/src/components/RightPane/SectionTable/cells/action.tsx
Comment (MinhxNguyen7) [inline apps/antalmanac/src/components/RightPane/SectionTable/cells/action.tsx:1]: I'd prefer to call the file `ActionCell` instead of just `action`, especially because it sounds like it's a module of form actions.

File: apps/antalmanac/src/components/RightPane/SectionTable/cells/action.tsx
Comment (MinhxNguyen7) [inline apps/antalmanac/src/components/RightPane/SectionTable/cells/action.tsx]: Considering this is only used once in `CopyAndAlert`, I think putting this down there would make it more readable instead of extracting it far away, especially because its name isn't strongly indicative of its *reason to-be*.

File: apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody.tsx
Comment (MinhxNguyen7) [inline apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody.tsx:490]: There's a type error here because the action cell is not currently a `SectionTableColumn`.

**Review by MinhxNguyen7 — CHANGES_REQUESTED** [review]
The action cell does not show up. Otherwise, only nitpicks.

image

**Comment by ap0nia** [comment]
My mistake, the columns not showing and the type errors were caused by me forgetting to commit the changes I made to the column store.

File: apps/antalmanac/src/components/RightPane/SectionTable/cells/action.tsx
Comment (ap0nia) [reply apps/antalmanac/src/components/RightPane/SectionTable/cells/action.tsx:1]: I disagree because the `action.tsx` file is in the `cells` directory, and accessing the file `cells/action-cell.tsx` seems redundant for the same way that you wouldn't create component files like `components/TutorialComponent.tsx`.

Comment (MinhxNguyen7) [reply apps/antalmanac/src/components/RightPane/SectionTable/cells/action.tsx:1]: Alright

**Review by MinhxNguyen7 — APPROVED** [review]
LGTM, but @KevinWu098 thinks we should wait until after the fee deadline to merge.

**Pull request merged**
